### ISSUE TITLE
Import Error: reason: image not found

### ISSUE THREAD

**Issue by songxiang666** [fbrundu/pymrmr#348]
* PymRMR version: 0.1.8
* Python version: 3.6.7
* Operating System:Mac OS 10.14.5

### Description
ImportError: dlopen(/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pymrmr.cpython-36m-darwin.so, 2): Library not loaded: /usr/local/opt/libiomp/lib/libiomp5.dylib
  Referenced from: /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pymrmr.cpython-36m-darwin.so
  Reason: image not found

### What I Did
I have installed libomp and llvm, but it does not work for me.
```
brew install libomp
brew install llvm
```
thanks in advance!

**Comment by test1855** [comment]
Same except for Operating System: Mac OS 10.12.3

And when I installed llvm, an error occurs.
Error: You are using macOS 10.12.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or IRC. You are responsible for resolving any issues you
experience while you are running this old version.

is there any way to sovle the Import Error and the llvm installing error in MacOS 10.12?

**Comment by test1855** [comment]
@songxiang666 
same problem! I have updated my Mac OS to 10.15.3 and updated brew, xcode, finally run "brew install llvm, libomp". The most important is to rename the folder 'libomp' in '/usr/local/opt/' to 'libiomp'  And the file 'libomp.dylib' in '/usr/local/opt/libomp/' to 'libiomp5.dylib'
It works for me. Hope it will help you.

**Comment by songxiang666** [comment]
@test1855 
Thank you for your help!
However, I still encountered problems.
I installed llvm successfully, so I think my system without any problems.
I also tried to update _Homebrew_.
The difference is that I don't have _xcode_ installed. (Do I have to install xcode?)
I also tried changing the directory name and file name, but the error I got was:

> Reason: no suitable image found.  Did find:/usr/local/opt/libiomp/lib/libiomp5.dylib: stat() failed with errno=20

Did you modify the file without any errors?

**Comment by test1855** [comment]
Yep, homebrew will failed if xcode is not installed. You can complete updating homebrew first.
